.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,42 @@
+name: ci
+
+on: [pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [14.x, 16.x, 18.x, 20.x]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: setup git
+        if: matrix.os == 'windows-latest'
+        run: |
+          git config core.symlinks true
+          git reset --hard
+
```

Comment:
Question here: Since `matrix.os= [ubuntu-latest]` (line 9 of this file), this should have no effect. Maybe we can remove it? Or is it added intentionally?